src/patternfly/components/Button/button.scss

```diff
@@ -316,6 +334,59 @@
     }
   }
 
+  &.pf-m-control {
+    &::after {
+      --pf-c-button--BorderWidth: var(--pf-c-button--m-control--after--BorderWidth);
+      --pf-c-button--BorderColor: var(--pf-c-button--m-control--after--BorderTopColor) var(--pf-c-button--m-control--after--BorderRightColor) var(--pf-c-button--m-control--after--BorderBottomColor) var(--pf-c-button--m-control--after--BorderLeftColor);
+
+      border-radius: unset;
```

Comment:
Any reason we wouldn't just use `0` here? We use `unset` very sparingly, and in this context, while it will still evaluate to `initial` (which is `0` for this property) we could just set that directly since we don't want it to have a `border-radius`.